uwblueprint/community-fridge-kw
# feat: Scheduling - Volunteer Information page

## Notion ticket link

[Scheduling Flow - Volunteer Information](https://www.notion.so/uwblueprintexecs/Team-Hub-5b0b225bfc09415ebcfad3030ce5fd30?p=8b1891e190fd4e62bebbb975494e56c2)



## Implementation description
* volunteer information page components
* pickup location does not support autofill atm




## Steps to test
1. For radio button components, test that the value is correctly updated on click.
2. For input/textarea components, type in the components and check that the strings are showing correctly in the boxes.



## What should reviewers focus on?
* 


## Checklist
